src/components/Rankings.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { getAllFieldCoveredCounts } from '../lib/coverageStats/statsUtils';
+
+const SectionColors = {
+  Patient: 'fill-[#D24200]',
+  Outcome: 'fill-[#8A45D9]',
+  Disease: 'fill-[#F2B84B]',
+  Treatment: 'fill-[#04B2D9]',
+  Assessment: 'fill-[#F2913D]',
+  Genomics: 'fill-[#26C485]',
+};
+const MINNUMSHOWN = 5;
+
+function Rankings({ coverageData }) {
+  const fields = getAllFieldCoveredCounts(coverageData).sort((a, b) => a.percentage - b.percentage);
+  const [numShown, setNumShown] = useState(MINNUMSHOWN);
+  const [buttonText, setButtonText] = useState(`See All ${fields.length}`);
+
+  function toggleRankingsShown() {
+    if (numShown === MINNUMSHOWN) {
+      setNumShown(fields.length);
+      setButtonText('Hide');
+    } else {
+      setNumShown(MINNUMSHOWN);
+      setButtonText(`See All ${fields.length}`);
+    }
+  }
+
+  return (
+    <div>
+      <h1 className="font-bold text-xl pb-3">Rankings</h1>
+      <table className="w-1/3">
```

Comment:
Let's also add padding to each row in our table. Since there isn't an easy way to do this at the `<table>` or `<tr>` level, we need to add a padding to each `<td>` element directly. Let's go with `py-1` padding.